### ISSUE TITLE
Some apps don't appear due to missing icon property (e.g. Netflix, Pandora)

### ISSUE THREAD

**Issue by jebrosen** [File-New-Project/EarTrumpet#45]
In EarTrumpet.Interop/AudioSessionService.cpp:28, retreiving both `PKEY_AppUserModel_Icon` and `PKEY_AppUserModel_PackageFullName` fail for some apps and they are excluded from the popup.

After commit jebrosen@5ed2aed195ad778686341803f0f95ea2596fd8a1 I am still able to control the volume of the apps with no icon:
![ear_trumpet_2016-01-01_21-09-32](https://cloud.githubusercontent.com/assets/7102588/12073182/04d78e90-b0cc-11e5-8e4d-e5d3dc6e2ec5.png)

The issue first appeared within the last week or so, but I have no idea what's different about the affected Apps or how to find when they were last updated. These apps obviously have some kind of icon as they appear in the start menu, but I read through the entire property list for these apps using `PSGetNameFromPropertyKey` and didn't see any mention of icons or logos.

**Comment by riverar** [comment]
Is this Pandora from the Store? Or some desktop app? Also, what build of Windows 10 are you on? Just trying to nail down a repro, thanks.

**Comment by jebrosen** [comment]
This is the Pandora app from the Store, yes. I'm on build 10586.36

To clarify, I've tried Pandora, Netflix, and two older Store games. Pandora and Netflix don't return icons (and I know Netflix updated recently). The older Store games still show their icons; Pandora and Netflix used to and don't anymore.

**Comment by jebrosen** [comment]
I think I've found the cause of the issue, and it's not actually a problem with Ear Trumpet. I recently pinned Pandora and later Netflix to the Classic Shell start menu. Windows was finding Classic Shell's "pinned" app shortcuts from the Start Menu folder and listing them in the Applications known folder. So there were two nearly identical Pandora shortcuts with the same name in the Applications known folder: Classic Shell's pin (located in the root "Start Menu" folder) and the original. Ear Trumpet was picking up Classic Shell's shortcuts which were missing the standard application properties.

Ironically, it was only because of Classic Shell I noticed the duplication issue: typing Pandora in the search box listed both shortcuts. The specific choice of recently updated applications that I look for in Ear Trumpet was only a coincidence.

The fix, ultimately, was moving Classic Shell's pin-storage from the start menu to a dedicated folder using the Classic Shell Settings page.
